.travis.yml

```diff
@@ -12,12 +12,40 @@ matrix:
         - python: "3.4"
         - python: "pypy3"
         - python: "pypy"
+before_install:
+    - pwd
+    - ls ~
+
+    - sudo apt-get update
+    - sudo apt-get -y install build-essential cmake libtool libwebp-dev openexr unzip python-pip python-dev
+    - pip install setuptools
+
+    - cd ~
+    - wget https://github.com/uclouvain/openjpeg/archive/version.2.0.1.tar.gz
+    - tar xf version.2.0.1.tar.gz
+    - cd openjpeg-version.2.0.1
+    - cmake .
+    - make
+    - sudo make install
+    - sudo make clean
+
+    - sudo ldconfig /usr/lib
+
+    - cd ~
+    - wget https://pypi.python.org/packages/source/P/Pillow/Pillow-2.9.0.zip#md5=cd4e6286fb28e277954c011c3ce05bc4
+    - unzip Pillow-2.9.0.zip
+    - cd Pillow-2.9.0
+    - python setup.py install
```

Comment:
@guilhermef Thanks!

I feel like I'm hitting a wall here however. I have not idea why tests are now failing for everything except the jpeg2000 tests. Any idea? https://travis-ci.org/thumbor/thumbor-plugins/jobs/81117784